consensus/src/member.rs

```diff
@@ -396,10 +397,13 @@ where
 /// For a detailed description of the consensus implemented by `run_session` see
 /// [docs for devs](https://cardinal-cryptography.github.io/AlephBFT/index.html)
 /// or the [original paper](https://arxiv.org/abs/1908.05156).
+#[allow(clippy::too_many_arguments)]
```

Comment:
I added something that will contain all local IO (so all IO without Network) and named it `LocalIO`. It is something like:
```
#[derive(Clone)]
pub struct LocalIO<D: Data, DP: DataProvider<D>, FH: FinalizationHandler<D>, UB: Write, UR: Read> {
    pub data_provider: DP,
    pub unit_backup: UB,
    pub unit_reader: UR,
    pub finalization_handler: FH,
    _phantom: PhantomData<D>,
}
```
So `Member::run_session(...)` takes such arguments
```
    config: Config,
    local_io: LocalIO<D, DP, FH, UB, UR>,
    network: N,
    keybox: MK,
    spawn_handle: SH,
    mut exit: oneshot::Receiver<()>,
```
I guess similar solution would be to add `Network` to it and have just `IO` containing those things. One thing worth mentioning is that `runway::run(...)` does not take `Network` (which it should not take), but takes all other thing. So in case it was `IO` we would have to somehow parameterize it by `Network`, or create another config for runway, not to have too_many_arguments there. What do you think about it @fixxxedpoint ?